home/views.py

```diff
@@ -58,20 +59,30 @@ def dashboard(request):
     query_date_str = request.GET.get('d', '')
     query_section = request.GET.get('s', '')
 
+    day_dict = {1: 'Monday', 2: 'Tuesday', 3: 'Wednesday',
+                4: 'Thursday', 5: 'Friday', 6: 'Saturday', 7: 'Sunday'}
+
     if query_date_str and query_section:
         query_date = datetime.strptime(query_date_str, '%Y-%m-%d').date()
         query_day = query_date.strftime("%w")
+        print(query_section)
         calendar = Calendar.objects.filter(date=query_date)
+        subject_today = Schedule.objects.filter(
+            day=query_day, section__section_id=query_section).get()
```

Comment:
Why have you used `get()` here? Also the variable name `subject_today` is wrong here. Neither it gives us subject but the schedule instance, nor it is for necessarily "today".